### PR TITLE
Add profiles for mypaint & mypaint-ora-thumbnailer

### PR DESCRIPTION
**Pull request by rusty-snake**

### CODE REVIEW

File: etc/mypaint.profile [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+disable-mnt
+private-cache
+private-dev
+private-etc fonts,gtk-3.0,dconf
+private-tmp
+
+noexec ${HOME}
+noexec /tmp
+
+# vim:set syntax=sh:
```
Comment (SkewedZeppelin) [inline etc/mypaint.profile]: remove

File: etc/mypaint.profile [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+nosound
+notv
+nou2f
+novideo
+protocol unix
+seccomp
+shell none
+tracelog
+
+disable-mnt
```
Comment (SkewedZeppelin) [inline etc/mypaint.profile]: remove
not in krita or gimp
users might store work on another drive

File: etc/mypaint.profile (hunk 1 above)
Comment (rusty-snake) [reply etc/mypaint.profile]: Upps, forgotten to remove.

**Comment by SkewedZeppelin** [comment]
Merged, thanks!

**Pull request merged**